nncore/utils/utils.py

```diff
@@ -40,7 +40,37 @@ def vprint(obj: str, verbose: bool):
         print(obj)
 
 
-def tensor2plt(obj: Tensor):
+def tensor2plt(obj: Tensor, title: List[Any]):
```

Comment:
https://matplotlib.org/3.1.1/api/_as_gen/matplotlib.pyplot.title.html

`title` should be `str`